class.jetpack.php

```diff
@@ -4023,7 +4023,11 @@ function build_connect_url( $raw = false, $redirect = false, $from = false ) {
 			    'admin.php?page=jetpack-settings' ), $url );
 			}
 		} else {
-			require_once JETPACK__GLOTPRESS_LOCALES_PATH;
+			if ( defined( 'JETPACK__GLOTPRESS_LOCALES_PATH' ) && file_exists( JETPACK__GLOTPRESS_LOCALES_PATH ) ) {
```

Comment:
> why file_exists instead of is_file && is_readable

Yeah, I generally prefer `is_file` too, since `file_exists` returns `true` for directories. In this case I just used the same conditions that other Jetpack modules used.

> what happens if the require fails? 

Yeah, it'll throw a fatal error. I generally prefer `require` over `include` because a missing/unreadable file indicates something is wrong, and I'd rather that be obvious so it can be fixed, instead of failing silently and going unnoticed. I can see how that wouldn't be the best approach for Jetpack, though, since it runs in such a wide array of hosting environments, etc.

Having the locale isn't critical, so we can definitely switch it to use `include_once`. The rest of the code is already setup to fall back to an empty string if the locale isn't set.

> Maybe it can be written like this...

I think that would throw a fatal error as well, but changing it to an `include_once` would allow for that approach:

```php
if ( defined( 'JETPACK__GLOTPRESS_LOCALES_PATH' ) && include_once JETPACK__GLOTPRESS_LOCALES_PATH ) {
  $gp_locale = GP_Locales::by_field( 'wp_locale', get_locale() );
}
```

Does that look good to you? If so, I'll update the PR.